src/mixins/render-vega-lite/Definitions/Mark/CrossSectionTerrainConfigDefinitionObject.js

```diff
@@ -0,0 +1,76 @@
+import ConfigDefinitionInterface from "../Config/ConfigDefinitionInterface"
+
+export default class CrossSectionTerrainConfigDefinitionObject extends ConfigDefinitionInterface {
+  // eslint-disable-next-line no-undef
+  static key = "lines"
+
+  // eslint-disable-next-line no-undef
+  static defaults = {
+    strokeColor: "black",
+    strokeWidth: 1,
+    fillBelowLine: 1,
+    opacity: 1,
+    fillOpacity: 1
+  }
+
+  /**
+   * @param {Object} definition_object
+   * @param {ParentInfo} parent_info
+   */
+  constructor(definition_object, parent_info) {
+    super(definition_object, parent_info)
+
+    this.strokeColor_ =
```

Comment:
If you didn't want to, we don't have to have all of this shenanigans to try and make these "private". It could just set the property on `this` and remove the `get` stuff below: `this.strokeColor = definition_object.strokeColor`